package.json

```diff
@@ -17,57 +17,54 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=18"
+    "node": "^18 || ^20"
   },
   "dependencies": {
     "@alaskaairux/auro-loader": "^1.1.1",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.4",
```

Comment:
These should be peer dependencies. Should be like 

```
  "peerDependencies": {
    "@aurodesignsystem/design-tokens": "",
    "@aurodesignsystem/webcorestylesheets": ""
  },
  "devDependencies": {
    "@aurodesignsystem/design-tokens": "",
    "@aurodesignsystem/webcorestylesheets": "",
    "@aurodesignsystem/eslint-config": "",
```

Reply:
What is `@aurodesignsystem/eslint-config`? Was that a package I was supposed to add?